app/views/spree/admin/payments/source_forms/_braintree.html.erb

```diff
@@ -0,0 +1,64 @@
+<fieldset data-id='credit-card' class="no-border-bottom">
```

Comment:
I guess maybe this would be an appropriate time to use hosted fields on both the front and backend?